data/commodities.txt

```diff
@@ -322,7 +322,7 @@ trade
 		"aircraft hulls"
 		"alarm systems"
 		"bicycles"
-		"breadmaking machines"
+		"bread making machines"
```

Comment:
```suggestion
		"breadmaking machines"
```
[breadmaking](https://en.wiktionary.org/wiki/breadmaking) appears to be accepted usage. It's not in the major dictionaries, but there are millions of uses of this term on Google. And you have publications the likes of the New York Times [[1]](https://www.nytimes.com/2022/09/12/dining/challah-recipe.html) [[2]](https://www.nytimes.com/2015/04/12/nyregion/at-razza-pizza-artigianale-in-jersey-city-taking-care-of-a-simple-dish.html?searchResultPosition=10) using this term as well, so I guess it's okay.